packages/graphile-build-pg/src/plugins/PgIntrospectionPlugin.js

```diff
@@ -57,6 +58,15 @@ export default (async function PgIntrospectionPlugin(
 
       const introspectionResultsByKind = rows.reduce(
         (memo, { object }) => {
+          if (object.description) {
+            if (pgEnableTags) {
+              const parsed = parseTags(object.description);
+              object.tags = parsed.tags;
+              object.description = parsed.text;
+            } else {
+              object.tags = {};
+            }
+          }
```

Comment:
This is a bit of a jammy place to add this code. It's not really the "proper" place; but I'll let it slide since it gets the job done - we can refactor later.